index.html

```diff
@@ -51,50 +51,57 @@
     <div class="container main-content" >
         <!-- row 1 -->
         <div id="content-search" class="row">
-            <div class="col-sm-4">
-                Text in row 1 column 1
+            <div id="row1-column1" class="col-sm-4">
+                <img id="card-image" src="">
+                
             </div>
-            <div class="col-sm-4" >
-    
+            <div id="row1-column2" class="col-sm-4" >
+              <div id="card-name-after-searched"> </div>
+              <div id="card-type-after-searched"> </div>
+              <div id="oracle-text"> </div>
+              <div id="power-and-toughness"> </div>
+
                 <!--Search bar-->
                     <form id="search-container" class="d-flex">
```

Comment:
So I did a search for teysa in the inital search box.  Then did a search for sol ring in the header search,  and the code just kept appending card name results to the <UL>.    It's easy enough to write a "clear" function to clear the UL search results before the search code is called.  

![image](https://user-images.githubusercontent.com/651674/126731900-040e3902-abe4-4db5-9865-25aa65f08a87.png)

Reply:
I would add more padding (aka negative space) between each Card Detail Item.    And just bring the flavor text up to the same row and the other details.

Finally capitalize the **R**arity.  

![image](https://user-images.githubusercontent.com/651674/126732113-bda0fc67-0c7c-4465-bb72-63a279563e12.png)